We.Sell.Bread.API/Controllers/CustomerController.cs

```diff
@@ -95,4 +95,29 @@ public async Task<ActionResult> DeleteCustomerAsync(string customerId)
 
         return IsdeletionSuccessful == false ? NotFound() : NoContent();
     }
+
+
+
+    [HttpPut(Name = "UpdateCustomer")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<ActionResult<CustomerDetailsDto>> Put(string customerId, NewCustomerDto customer)
+    {
+        _logger.LogInformation($"Updating the following customer's details: {customer.CustomerName}");
```

Comment:
Can you you please return bad request if the customerId is also not a valid GUID.